jmx/src/test/java/org/jboss/as/jmx/PluggableMBeanServerTestCase.java

```diff
@@ -105,6 +112,33 @@ public void testReservedDomainMBeanRegistrationFails() throws Exception {
         reservedDomainTest("jboss.as.expr:bean=test-null", NAME);
     }
 
+    @Test
+    public void testTccl() throws Exception {
```

Comment:
Minor: can you call this "testNotificationTccl"?  Or something like that.

Reply:
@bstansberry  I've changed it to testNotificationTccl